include/picongpu/particles/flylite/NonLTE.tpp

```diff
@@ -29,7 +29,7 @@
 
 #include <pmacc/Environment.hpp>
 #include <pmacc/dataManagement/ISimulationData.hpp>
-#include <pmacc/traits/GetNumWorkers.hpp>
+#include <pmacc/lockstep/lockstep.hpp>
```

Comment:
Is this include even required here? Considering there are no other changes in this file, it could that `GetNumWorkers.hpp` was also superfluous.

Reply:
I removed it, it was wrong before and auto substitution changed it.